core/src/com/mygdx/game/ecs/GameEngine.java

```diff
@@ -41,4 +48,24 @@ public void initializeEngine() {
         gameEngineInstance.addSystem(new BotControlSystem());
         gameEngineInstance.addSystem(new HealthRenderSystem());
     }
+
+    private void createBackground() {

```

Comment:
Kan denne være hete noe med  factory?

Reply:
Tror ikke det gir mening å kalle en metode her for en factory, i hvert fall ikke når den bare lager en spesifikk entity. 

For Bots gir det veldig mening, ettersom det skal være mange bots, og potensielt flere typer. 
For player er jeg litt usikker, det er vel nesten i grenseland om den trenger å være factory i det hele tatt, men den er litt mer kompleks enn background, så kan være greit sånn sett. 
Slik som createBackground er satt opp nå tror jeg det blir det en veldig stretch å kalle den for en factory.

Reply:
Den er god :D